lua/rest-nvim/request/init.lua

```diff
@@ -174,7 +174,7 @@ local function end_request(bufnr)
   end
   utils.move_cursor(bufnr, linenumber)
 
-  local next = vim.fn.search("^GET\\|^POST\\|^PUT\\|^PATCH\\|^DELETE", "cn", vim.fn.line("$"))
+  local next = vim.fn.search("^GET\\|^POST\\|^PUT\\|^PATCH\\|^DELETE\\|^//\\|^#", "cn", vim.fn.line("$"))
```

Comment:
I don't think parsing blocks based on comments is a good idea, as:
1. it goes against rfc.
2. comments by design shouldn't interact with the code.